src/modules/user/components/about-professional-form/about-professional-form.tsx

```diff
@@ -140,33 +144,37 @@ const AboutProfessionalForm = memo<AboutProfessionalFormProps>(
             }}
           />
         </div>
-        <div className={styles.inputsRow}>
-          <TextField
-            {...form.register('facebook')}
-            error={form.formState.errors.facebook?.message}
-            variant='input'
-            label={intl.formatMessage({
-              id: 'user.about.professional.form.facebook',
-            })}
-          />
+        {me?.userType === 'PROFESSIONAL' && (
```

Comment:
It seems like you do this `me?.userType === 'PROFESSIONAL` two times, can you mrge them into one check?